domino/airflow/_operator.py

```diff
@@ -99,15 +102,18 @@ def execute(self, context: Any) -> dict:
             poll_freq=self.poll_freq,
             max_poll_time=self.max_poll_time
         )
+        self.client._logger.setLevel(logging.INFO)
```

Comment:
same here

Reply:
Changed to use a property on the main class: `log`